docs/source/algorithms/XrayAbsorptionCorrection-v1.rst

```diff
@@ -0,0 +1,65 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm calculates the correction factors due to the absorption of Xrays 
+by the sample and applies it to Input Workspace. This is done by determining path of an xray from a 
```

Comment:
Does the algorithm apply the correction to the input workspace? It looks to me as though the algorithm calculates the correction factor and then (in line with the other absorption correction algorithms) the user is responsible for applying the correction afterwards - this is normally done by dividing a real "signal" workspace by the calculated correction factor.

Reply:
It does not apply the correction it would have to be done by the user

Reply:
The reason I asked is that the first sentence I've attached my comment to says "...and applies it to the Input Workspace". So I think this text needs to be removed?